contracts/interfaces/access-control/IMultiOwnable.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.4;
+
+interface IMultiOwnable {
+    event OwnerAdded(address[] newOwners);
+    event OwnerRemoved(address[] removedOwners);
+
+    /**
+     * @notice Owner can add new owners to the _owners.
```

Comment:
```suggestion
     * @notice Owner can add new owners to the contract's owners list.
```

Reply:
The thing is these comments are meant to be presented to the users of this contract. The notice description should be user-focused, not implementation-focused